src/olympia/templates/photon-footer.html

```diff
@@ -14,7 +14,7 @@ <h4 class="Footer-links-header"><a href="{{ url('home') }}">Add-ons</a></h4>
                 <li><a href="{{ url('devhub.index') }}">Developer Hub</a></li>
                 <li><a href="https://developer.mozilla.org/docs/Mozilla/Add-ons/AMO/Policy">Developer Policies</a></li>
                 <li><a href="https://discourse.mozilla-community.org/c/add-ons">Forum</a></li>
-                <li><a class="Footer-bug-report-link" href="https://developer.mozilla.org/docs/Mozilla/Add-ons/Contact_us">Report a bug</a></li>
+                <li><a class="Footer-bug-report-link" href=" https://addons.mozilla.org/about">Report a bug</a></li>
```

Comment:
```suggestion
                <li><a class="Footer-bug-report-link" href="https://addons.mozilla.org/about">Report a bug</a></li>
```

Reply:
I am sorry xDDD
Will fix it right away

Reply:
You can probably apply the suggestion I made.

Reply:
Oh I didn't know about this feature